claim_from_delivery/readme/CONTRIBUTORS.rst

```diff
@@ -0,0 +1,4 @@
+* Stefan Rijnhart <stefan@opener.amsterdam> (https://opener.amsterdam)
```

Comment:
I didn't do anything! You can remove me from this bit

Reply:
Would be nice if you could list the committers on the module from the Odoo tree though.

Reply:
CREDITS.rst is an empty file below, so Github won't let me comment on it, but can you add the disclaimer from https://github.com/OCA/crm/commit/8078fc9601bd11fb9db060d8e0a5fbcfd006facf#diff-7a2d2e4fa90e6191ec005372e4b17018R47-R49?